### ISSUE TITLE
Sec gas masks worn "out of the way" still blind you, despite no protection, sprite only covers your mouth, yet appears to cover the face in code

### ISSUE THREAD

**Issue by Robustitron** [tgstation/tgstation#63264]
## Round ID:
[174218](https://scrubby.melonmesa.com/round/174218)

Security gas masks, visually, appear to be fancier versions of air masks, that cover your mouth, with the addition of voice lines.

No one wears them recently, because, bizzaringly, they blind your eyes without visually covering them, this stacks with sec glasses, which sec always wears, making them infeasible for sec, it's optimal to drop them and find air masks in a locker.

Even when the masks are worn "out of the way", no longer covering your mouth, as well as no longer protecting your eyes from pepper spray, they still manage to be able to blind your eyes, that they're not actually covering, or protecting.

Any eye or pepper spray protection given by security gas masks is also nonsensical, because the sprite never covered the eyes, I always thought this was a bug.

TL;DR, the security gas mask look broken, and if it is working by design, it's very poorly implemented, since it's behavior is the opposite of what the sprite implies. A mask that only covers your mouth is somehow protecting your eyes and also blinding them, moving the mask out of the way stops the eye protection, but doesn't stop the eye blinding.

**Comment by Dorsisdwarf** [comment]
This should have been fixed by #63211, can you reproduce this @Robustitron

**Comment by PolishUser** [comment]
Ghilker not testing his code, what a shocker!

**Comment by SuperNovaa41** [comment]
I'm fairly certain that this is an issue with clothing code as a whole. Adjusting and readjusing causes weird things with clothing flags.

**Comment by Robustitron** [comment]
I went and tested, it's fixed, sec mask has no tint, and no pepper spray protection regardless of how it's worn.

The problem existed the day before, my fault was posting this bug a day late, not expecting it to be fixed, and just attaching the latest server revision I saw on bagil, which wasnt accurate.
